api/interestr/api/models.py

```diff
@@ -133,5 +133,19 @@ class DataTemplate(BaseModel):
     def __str__(self):
         return self.name
 
+class AnnoTypes(ChoiceEnum):
+    text = "Text"
+    image = "Image"
+
+class Annotation(BaseModel):
+    anno_id = models.AutoField(primary_key=True)
+    user = models.ForeignKey(auth_models.User, related_name='%(app_label)s_%(class)s_related', on_delete=models.SET_NULL, default=None,
```

Comment:
What is `'%(app_label)s_%(class)s'` here?

Reply:
Sorry. Before discussing with you, I created this class as an abstract class which was meant to be inherited from by text and image annotation classes. Thus, different lookup names were required for subclasses.